### PR TITLE
Only log events on congrats and certificates pages for teachers

### PR DESCRIPTION
**Pull request by bethanyaconnor**
Dani pointed out that these events are sending for all users. Because we do have an event quota with Amplitude, I'm updating this to only send events for teachers.

### CODE REVIEW

File: apps/src/sites/studio/pages/print_certificates/show.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
 import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
 import {EVENTS} from '@cdo/apps/lib/util/AnalyticsConstants';
+import {getStore} from '@cdo/apps/redux';
 
 $(() => {
-  analyticsReporter.sendEvent(EVENTS.CERTIFICATE_PRINT_PAGE_VISITED);
+  const isTeacher = getStore().getState().currentUser.userType === 'teacher';
+  if (isTeacher) {
+    analyticsReporter.sendEvent(EVENTS.CERTIFICATE_PRINT_PAGE_VISITED);
+  }
```
Comment (kelbyhawn) [inline apps/src/sites/studio/pages/print_certificates/show.js:9]: Learning Q: is this how you add Amplitude events on studio.code.org? (new logic notwithstanding)

File: apps/src/sites/studio/pages/print_certificates/show.js [hunk 2]
```diff
@@ -1,6 +1,10 @@
 import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
 import {EVENTS} from '@cdo/apps/lib/util/AnalyticsConstants';
+import {getStore} from '@cdo/apps/redux';
 
 $(() => {
-  analyticsReporter.sendEvent(EVENTS.CERTIFICATE_PRINT_PAGE_VISITED);
+  const isTeacher = getStore().getState().currentUser.userType === 'teacher';
```
Comment (kelbyhawn) [inline apps/src/sites/studio/pages/print_certificates/show.js:6]: Learning Q: is this a good example of using data coming from the database?

**Review by kelbyhawn — APPROVED** [review]
Have a couple learning questions, but otherwise looks good! Someone else might want to take a look too though.

File: apps/src/sites/studio/pages/print_certificates/show.js (hunk 1 above)
Comment (hannahbergam) [reply apps/src/sites/studio/pages/print_certificates/show.js:9]: yuuup it's v simple on dashboard!

**Review by hannahbergam — APPROVED** [review]
This makes sense! Just to confirm: were you able to test via both account types in the console?

File: apps/src/sites/studio/pages/print_certificates/show.js (hunk 2 above)
Comment (bethanyaconnor) [reply apps/src/sites/studio/pages/print_certificates/show.js:6]: This is more of an example of getting data from Redux (which, in this case, does come from the database at some point). But, definitely an example of a case where this page will act differently for different users!

**Comment by bethanyaconnor** [comment]
> Just to confirm: were you able to test via both account types in the console?

Yep! Tested for a teacher (event fired), as well as signed out and for a student (no event fired).

File: apps/src/sites/studio/pages/print_certificates/show.js (hunk 2 above)
Comment (kelbyhawn) [reply apps/src/sites/studio/pages/print_certificates/show.js:6]: Cool, ty!

**Pull request merged**